addons/l10n_eg_edi_eta/models/account_edi_format.py

```diff
@@ -68,15 +69,16 @@ def _l10n_eg_eta_connect_to_server(self, request_data, request_url, method, is_a
                 'error': str(ex),
                 'blocking_level': 'warning'
             }
-        if not request_response.ok:
```

Comment:
Why is this condition removed?

Reply:
Hello @jco-odoo sorry for the mistake. Thank you for noticing it :) . I have updated the code please review it